CITATION.cff

```diff
@@ -0,0 +1,9 @@
+cff-version: 1.2.0
+title: "torchtune: the PyTorch native finetuning library"
```

Comment:
```suggestion
title: "torchtune: a PyTorch native finetuning library"
```

Reply:
You don't like being THE finetuning library?? 🥹

Reply:
Or we can split the difference and say "PyTorch's finetuning library"